### PR TITLE
link acr

### PR DESCRIPTION
**Pull request by jaiveerk**
linking for msi clusters plus some groundwork for sp clusters

### CODE REVIEW

File: pkg/azure/aks.go [hunk 1]
```diff
@@ -57,6 +63,89 @@ func ListClusters(ctx context.Context, subscriptionId, resourceGroup string) ([]
...
+
+	if err != nil {
+		return fmt.Errorf("getting cluster information: %w", err)
+	}
+
+	var assigneeId *string
+
+	if cluster.Identity == nil {
+		return fmt.Errorf("serviceprincipal clusters are not supported at this time")
+		//lgr.Debug("detected service principal cluster")
```
Comment (davidgamero) [inline pkg/azure/aks.go:90]: should we remove the commented code?

File: spin-aks-plugin
Comment (davidgamero) [inline spin-aks-plugin:1]: gotta remove this binary

**Review by davidgamero — COMMENTED** [review]
wow you really fought to make this happen!

File: pkg/azure/aks.go (hunk 1 above)
Comment (jaiveerk) [reply pkg/azure/aks.go:90]: want to leave it for if/when we need to support sp clusters

File: spin-aks-plugin
Comment (jaiveerk) [reply spin-aks-plugin:1]: done!

**Review by davidgamero — APPROVED** [review]
lgtm

**Pull request merged**